.bazelrc

```diff
@@ -57,7 +57,9 @@ build:secure --config=relro
 # =================================
 # Release
 # =================================
-build:release --compilation_mode opt
+build:release --copt -O2
+build:release --copt -g0
+build:release --copt -DNDEBUG
```

Comment:
* this commit doesn't touch forify_source, but it's discussed in the commit message.
* why don't we use `opt` compilation mode? i'd assume it adds -02 and NDEBUG. is that not the case?
* what is g0?

Reply:
i guess i'm just wondering why we are removing `opt` -- it seems to add this stuff in for us.

Reply:
The `--compilation_mode opt` sets a little bit more than we want it to. For example it sets the `--copt -D_FORTIFY_SOURCE=1` which we do not want.